lms/templates/bookmark_button.html

```diff
@@ -0,0 +1,11 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%page args="is_bookmarked, bookmark_id" />
+
+<div class="bookmark-button-wrapper">
+  <button class="btn bookmark-button ${"bookmarked" if is_bookmarked else "un-bookmarked"}"
+    aria-pressed="${"true" if is_bookmarked else "false"}"
+    data-bookmark-id="${bookmark_id}">
+     <span class="sr bookmark-sr">${_("Click to remove bookmark from this unit") if is_bookmarked else _("Click to bookmark this unit")}</span>
```

Comment:
I like this, thanks for providing extra context.  However, I would simplify it a bit.  Its redundant:
"Click to remove bookmark from this unitBookmark"  The visible text is also included in the accessible name.
how about:
```<span class="sr bookmark-sr">${_("Click to remove") if is_bookmarked else _("Click to")}</span>&nbsp;```
The word "Bookmark" is already provided, the screen reader will say "Click to remove Bookmark", and "Click to Bookmark".
Also notice how i added a ```&nbsp;``` inside of the span.  Without it, the screen reader text would be 
"Click to removeBookmark"